drivers/sensor/lsm6dsl/Kconfig

```diff
@@ -146,6 +146,7 @@ config LSM6DSL_ACCEL_ODR
 	  8: 1666Hz
 	  9: 3332Hz
 	  10: 6664Hz
+	  11: 1.6Hz
```

Comment:
the `range` max value should be adapted, isn't it?

Reply:
> the range max value should be adapted, isn't it?

I'm not sure I understand. The range (or scale) and frequency of the accelerometer are independent of each other.

Reply:
Oh, right! Updating the `Kconfig`. Thanks.